### ISSUE TITLE
Rotate icons on a small screen

### ISSUE THREAD

**Issue by AlexRMU** [josdejong/svelte-jsoneditor#345]
![image](https://github.com/josdejong/svelte-jsoneditor/assets/61969794/0110fdfc-0382-422f-b6f3-4850563a1604)
---
![image](https://github.com/josdejong/svelte-jsoneditor/assets/61969794/89f09264-7674-41d1-b709-79df70306fdc)

**Comment by josdejong** [comment]
Good idea! I've made the actions panel more responsive and rotate the icons. I'll let you know when it is published online:

![afbeelding](https://github.com/josdejong/svelte-jsoneditor/assets/568626/fd542673-d491-4a05-9a5d-55cbf0139be6)

BTW what happened with the colors in your screenshots? They're not the original theme.

**Comment by AlexRMU** [comment]
dark reader :sunglasses:

**Comment by AlexRMU** [comment]
See how they differ:
| dark theme  | dark theme + dark reader |
| ------------- | ------------- |
| ![image](https://github.com/josdejong/svelte-jsoneditor/assets/61969794/1c2e58ae-a59f-4775-a316-295ff2d0911b) | ![image](https://github.com/josdejong/svelte-jsoneditor/assets/61969794/e423da69-011b-40d0-bef1-4c768f99f688) |
